.changeset/breezy-carrots-bow.md

```diff
@@ -0,0 +1,8 @@
+---
+"@fuel-ts/utils": patch
+"@fuel-ts/account": minor
+"@fuel-ts/contract": patch
+"fuels": patch
```

Comment:
Considering everything is bumped together, what's the utility in mixing `patch` and `minor`?